README.rst

```diff
@@ -8,39 +8,115 @@ Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
 
-  ``git clone https://github.com/edx/frontend-template-application.git``
+   .. code-block::
 
-2. Install npm dependencies:
+      DEFAULT_SERVICES ?= \
+      credentials+discovery+ecommerce+edx_notes_api+forum+frontend-app-publisher+frontend-app-learning+gradebook+lms+studio+frontend-app-library-authoring
 
-  ``cd frontend-template-application && npm install``
+   This adds ``frontend-app-library-authoring`` to the list of services the devstack will provision and enable automatically.
 
-3. Start the dev server:
+3. Proceed with the setup of the devstack as described in the README's `Getting Started section
+   <https://github.com/edx/devstack#getting-started>`_.
 
-  ``npm start``
+   When running ``make dev.clone``, `this repository <https://github.com/edx/frontend-app-library-authoring.git>`_ will
+   be cloned automatically as a the sibling to the `~/workspace/devstack/` directory.
 
-The dev server is running at `http://localhost:8080 <http://localhost:8080>`_.
+   Once devstack provisioning is complete, check that this MFE's container is running by executing the following from
+   inside ``~/workspace/devstack``:
+
+   .. code-block::
+
+      make dev.ps | grep library-authoring
+
+   You should see the container with status "Up", exposing port 3001:
+
+   .. code-block::
+
+      edx.devstack.frontend-app-library-authoring   docker-entrypoint.sh bash  ...   Up       0.0.0.0:3001->3001/tcp
+
+4. Now set up blockstore for this devstack exactly as described in its README's `Using with Docker Devstack
+   <https://github.com/edx/blockstore/#using-with-docker-devstack>`_ section.
+
+   This will setup the blockstore container, configure the LMS and the CMS to accept requests from it (and vice-versa),
+   create a "Devstack Content Collection" in blockstore, and finally create a sample "DeveloperInc" organization in the
+   LMS that can be used to create content.  Make a note of the UUID used to create the "Devstack Content Collection".
+   At the time of writing, the value suggested in the README is:
+
+   .. code-block::
+
+      11111111-2111-4111-8111-111111111111
+
+   There's no need to log in to blockstore in your web browser directly, so feel free to skip the last step.
+
+5. Edit ``frontend-app-library-authoring/.env.development`` and add the "Devstack Content Collection" UUID you obtained
+   above to ``BLOCKSTORE_COLLECTION_UUID``:
+
+   .. code-block::
+
+      BLOCKSTORE_COLLECTION_UUID=11111111-2111-4111-8111-111111111111
+
+   Restart the frontend-app-library-authoring container to apply the change:
+
+   .. code-block::
+
+      cd devstack
+      make dev.restart-container.frontend-app-library-authoring
+
+6. In a Studio shell, enable the ``ENABLE_LIBRARY_AUTHORING_MICROFRONTEND`` feature flag:
+
+   .. code-block::
+
+      make studio-shell
+      vim /edx/etc/studio.yml
+      ---
+      FEATURES:
+          ENABLE_LIBRARY_AUTHORING_MICROFRONTEND: true
```

Comment:
I think this pattern came from the `frontend-app-learning` toggling scheme, which I suggested we use as a template, not realizing that it's way overkill for this MFE.

+1 to picking one of the toggles and ditching the other when we get the chance. I prefer the Django setting because it's version controlled, but the Waffle is nice because you can enable it for individual users. Anything but both!